MarsQA/SpecflowPages/LanguagesPage.cs

```diff
@@ -1 +1,99 @@
-﻿
\ No newline at end of file
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium.Support.UI;
+using MarsQA.Drivers;
+using System.Collections.ObjectModel;
+
+namespace MarsQA.SpecflowPages
+{
+    public class LanguagesPage
+    {
+        private IWebDriver driver;
+
+        public LanguagesPage()
+        {
+            this.driver = CommonDriver.driver;
+        }
+
+        public void CreateLanguageRecord()
+        {
+            try
```

Comment:
Please implement the POM Pattern, please see the below  for example how it looks you can see the pages how they are set 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html